src/monitoring/pumpportal_listener.py

```
@@ -0,0 +1,166 @@
+                                logger.info(
+                                    f"Token not created by {creator_address}. Skipping..."
+                                )
+                                continue
+
+                            await token_callback(token_info)
+
+                    except websockets.exceptions.ConnectionClosed:
+                        logger.warning("PumpPortal WebSocket connection closed. Reconnecting...")
+                        ping_task.cancel()
+
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)